_support-articles/windows/special-characters

```diff
@@ -0,0 +1,19 @@
+---
+layout: article
+title: "Fixing crashes related to 'special' characters in Windows usernames"
+last_updated: 2019-01-28 13:00:00 +0000
+---
+
+Having an exclamation point (!), or some non-English Alphanumeric characters in your Windows Username (local windows user account) will cause the game to crash.
+
+In some cases you can fix this by changing the working directory the game uses by following the instructions [here](/support-articles/changing-working-direcotry/).
+
+If that does not resolve the issue, you will need to create a new local Windows username containing only English Alphanumernic characters.
+To be clear, this includes only (abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789) exactly as they are seen on this page.
+**NOTE:** Changing your Windows username will not correct the issue, you **NEED** to create a new local username.
+
+## Instructions for supported Windows Versions:
+
+Windows 7, 8, 8.1: https://support.microsoft.com/en-us/help/13951/windows-create-user-account#create-user-account=windows
```

Comment:
Windows 
XP - http://support.microsoft.com/kb/279783
Vista - http://windows.	microsoft.com/en-us/windows/create-user-account#create-user-account=windows-vista
7 - http://windows.microsoft.com/en-us/windows/create-user-account#create-user-account=windows-7
8/8.1 - http://windows.microsoft.com/en-us/windows/create-user-account#create-user-account=windows-8
10 - https://support.microsoft.com/en-us/help/4026923/windows-10-create-a-local-user-or-administrator-account
Mac
10.6 Snow Leopard - http://support.apple.com/kb/ph6651
10.7 Lion - http://support.apple.com/kb/ph4600
10.8 Mountain Lion - http://support.apple.com/kb/PH11468

Reply:
This one is windows only, xp and vista are unsupported, and the links for 7 8 8.1 are all to the same page

Reply:
We have the room, so I'd rather just link properly for the versions that matter. No need for a note to save a line.

Reply:
Its the same link for each version, the different numbered links make no change to how the page is shown. The note would still be needed for people that don't know how to read through things.

Reply:
The # part in the URL makes the page jump to the correct place

Reply:
It has not done that once for me.

Reply:
I'd have to agree with matt here, the URLs didn't redirect me properly.